gnirehtet.cmd

```diff
@@ -0,0 +1,150 @@
+@ECHO OFF
+CALL :locate_file ADB %~dp0adb.exe "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
```

Comment:
> There isn't a system var for adb in windows. I'm searching adb in the same directory or in default path of Android Studio.

I think that many people using Windows add the parent directory of `adb` in `%PATH%`, so that they can call `adb.exe` from everywhere, don't they?

Does `:locate_file` searches in `%PATH%` too?

The _bash_ version expects `adb` and `java` to be callable, so `$PATH` must be defined correctly for the script to work (the packages install them in directories defined in `$PATH`, so it works as expected by default).

IMO, the Windows version should also use `adb.exe` accessible from `%PATH%` first: if the user correctly defined its `%PATH%` to make `adb.exe` accessible from everywhere, the script should find it.

What do you think?